ksiren/build.gradle

```diff
@@ -5,10 +5,10 @@ def group = "com.brightspace.ksiren"
 def artifactName = "ksiren"
 def vcsUrlSetting = "https://github.com/Brightspace/ksiren/"
 def description = 'Main package of KSiren: A Kotlin library for parsing responses from Siren API endpoints and building Siren Action requests.'
-def versionValue = "1.1.0-beta3"
+def versionValue = "1.1.0"
```

Comment:
For Portfolio at least, we are using the internal KSiren... do these changes need to be duplicated there as well?

Reply:
These changes are just a port of the changes to the internal repo, minus the change to openjdk8 for travis, which should probably get ported.